### ISSUE TITLE
/apps/<appname>/processes API call hangs

### ISSUE THREAD

**Issue by mattmanning** [convox/rack#355]
```
$ curl -v "convox:<redacted>@convox-dev-128493365.us-east-1.elb.amazonaws.com/apps/feedchirp/processes"
*   Trying 54.175.204.251...
* Connected to convox-dev-128493365.us-east-1.elb.amazonaws.com (54.175.204.251) port 80 (#0)
* Server auth using Basic with user 'convox'
> GET /apps/feedchirp/processes HTTP/1.1
> Host: convox-dev-128493365.us-east-1.elb.amazonaws.com
> Authorization: Basic <redacted>
> User-Agent: curl/7.43.0
> Accept: */*
>
```

This never returns ^

**Comment by mattmanning** [comment]
More info: This returns if I add the query string `?stats=false` to the URL. Looks like stats=true is the default and is broken. `convox ps --stats` from the CLI is broken too.

**Comment by nzoschke** [comment]
I can't yet reproduce the hanging on testing racks, even with a ton of processes.

**Comment by bitwalker** [comment]
@nzoschke @xtfxme and I are seeing this on our racks, though I have yet to pin it down to a specific sequence of events - just wanted to at least let you know that we're seeing this as well. We don't have the latest from master on our racks yet (I'd have to double check with @xtfxme to know for sure), so it's possible this has already been addressed.

**Comment by nzoschke** [comment]
@awsmsrc and I are working on a fix here:

https://github.com/convox/rack/pull/394

The root cause appears to be the done channel hanging sporadically on the Docker client Stats.

I'll update close this ticket when the patch lands.

**Comment by nzoschke** [comment]
This is fixed and released in 20160302193502
